### PR TITLE
Johnson Algorithm in Python Fixes #2206

### LINKED ISSUE #2206: Johnson Algorithm
- [x] C 
- [ ] C++ - @Aakansha99
- [x] C# - @ritish099
- [ ] CoffeeScript
- [x] Dart - @Swarnimashukla
- [ ] Go - @BhardwajShweta25
- [ ] Java - @Neelaksh-Singh
- [ ] JS - @thakran14
- [x] Kotlin - @ritish099
- [x] Php - @ritish099
- [ ] Python - @BishalKrChanda
- [ ] Readme - @AdityaSingh17
- [ ] Ruby
- [ ] Rust
- [ ] TypeScript

### PR DESCRIPTION
**Pull request by BishalKrChanda**
Fixes #

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] Sample Input / Output is added at the end of file.
- [x] Logic Documentation (Comments).
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Addition of folder for Johnson Algorithm
- Addition of code for Johnson Algorithm in Python

#### Languages Used:
- Python

#### Files Added:
- Johnson_Algorithm.py

> We're happy to help you get this ready -- don't be afraid to ask for help.

Thanks!

### CODE REVIEW

File: Johnson_Algorithm/Johnson_Algorithm.py [hunk 1]
```diff
@@ -0,0 +1,220 @@
...
+    d[source] = 0
+ 
+    for _ in range(len(g) - 1):
+        for v in g:
+            for n in v.get_n():
+                d[n] = min(d[n], d[v] + v.get_w(n))
+ 
+    return d
+ 
+
```
Comment (SakshiShreya) [inline Johnson_Algorithm/Johnson_Algorithm.py]: Only 1 new-line should be there between two functions

File: Johnson_Algorithm/Johnson_Algorithm.py [hunk 2]
```diff
@@ -0,0 +1,220 @@
...
+        # Updating distances
+        for neighbour in closest.get_n():
+           if neighbour in unvisited:
+               new_d = d[closest] + closest.get_w(neighbour)
+               if d[neighbour] > new_d:
+                   d[neighbour] = new_d
+ 
+    return d
+ 
+
```
Comment (SakshiShreya) [inline Johnson_Algorithm/Johnson_Algorithm.py]: Remove this new-line too

File: Johnson_Algorithm/Johnson_Algorithm.py [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+        for v in g:
+            print(v.get_k(), end=' ')
+        print()
+ 
+        print('Edges: ')
+        for v in g:
+            for destination in v.get_n():
+                z = v.get_w(destination)
+                print('(source={}, destination={}, w={}) '.format(v.get_k(),
+                                                             destination.get_k(), z))
```
Comment (SakshiShreya) [inline Johnson_Algorithm/Johnson_Algorithm.py]: Why are there so many spaces here?

**Comment by somya-kapoor** [comment]
@SakshiShreya Please review

**Comment by BishalKrChanda** [comment]
@SakshiShreya @somya-kapoor please check this too. Its been more than half a month.

**Comment by BishalKrChanda** [comment]
@SakshiShreya @somya-kapoor @jainaman224  please check this. Its been 18 days.

**Comment by MastersAbh** [comment]
@jainaman224

**Pull request closed without merging**